examples/08_data_observatory/download.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Big Query Client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cartoframes.auth import Credentials\n",
+    "credentials = Credentials.from_file()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cartoframes.data.clients.bigquery_client import BigQueryClient\n",
+    "client = BigQueryClient('carto-do-customers', credentials)"
```

Comment:
I'd like this class to be as much hidden as possible and having it in an example may make it to be used... Could we just showcase the `Dataset` `Geography` download methods?

Reply:
:+1: You are right. This was util for development, but no now